stdpopsim/cli.py

```diff
@@ -354,7 +354,8 @@ def add_simulate_species_parser(parser, species):
                 f"Default={choices[0]}."))
     species_parser.add_argument(
         "-l", "--length-multiplier", default=1, type=float,
-        help="Simulate a chromsome of length l times the named chromosome")
+        help="Simulate a sequence of length l times the named chromosome, using "
```

Comment:
should this be `"Simulate a chromsome of length l times the named chromosome's length"`?